schemas/descriptors/label/descriptorLabel.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "meta:license": [
+    "Copyright 2021 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/descriptors/descriptorLabel",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Label Descriptor",
+  "meta:extensible": false,
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/common/desciptors/schemadescriptor",
+    "https://ns.adobe.com/xdm/common/descriptors/labeldescriptor"
+  ],
+  "meta:abstract": false,
+  "type": "object",
+  "description": "Describes a label at the field level for a given class/fieldgroup/schema",
+  "definitions": {
+    "descriptorLabel": {
+      "properties": {
+        "@type": {
+          "type": "string",
+          "const": "xdm:descriptorLabel",
+          "meta:xdmType": "string"
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/descriptors/schemadescriptor#/definitions/descriptor"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/common/descriptors/labeldescriptor#/definitions/labeldescriptor"
```

Comment:
No reason to create this external labeldescriptor file. Just add the "xdm:labels" field definition in the local definition here.

Reply:
Yes. My bad. There is no reason for that file.